.github/workflows/karate/be1-go.yaml

```diff
@@ -1,11 +1,11 @@
-name: Integration Tests
+name: Integration - Go
```

Comment:
Are you sure you can move this file in a subfolder?

Reply:
Well, you're right.

It does not seem to work